deploy/FlapperInit.sol

```diff
@@ -50,14 +50,14 @@ interface PairLike {
     function token1() external view returns (address);
 }
 
-interface DaiJoinLike {
-    function dai() external view returns (address);
+interface NstJoinLike {
```

Comment:
For consistency we should also rename `DaiJoinLike` > `NstJoinLike` in the Splitter (and SplitterMock).